commcare_connect/opportunity/tests/test_visit_import.py

```diff
@@ -324,10 +325,13 @@ def test_bulk_update_payments(opportunity: Opportunity):
 
     assert Payment.objects.filter(opportunity_access__opportunity=opportunity).count() == 5
 
-    for access in access_objects:
+    for index, access in enumerate(access_objects):
         payment = Payment.objects.get(opportunity_access=access)
         assert payment.amount == 50
-        assert payment.date_paid.strftime("%Y-%m-%d") == payment_date
+        if index == 4:
```

Comment:
instead of checking the index like this, the test would be more robust if we passed the uploaded data. it would also be easier to reason about, rather than trying to understand why 4 is a magic number. Ideally that would use something like the parametrized tests we use elsewhere, but if that is tricky with the rest of this test setup, storing the list from line 307 as you pass it to the dataset, would allow you to cross reference it here